hydroflow/src/scheduled/mod.rs

```diff
@@ -27,17 +31,19 @@ pub struct Hydroflow {
 
     // TODO(mingwei): separate scheduler into its own struct/trait?
     ready_queue: VecDeque<OpId>,
+    scheduled_operators: Vec<bool>,
```

Comment:
Can we include this as a field in `SubgraphData` for now?

Later we should think about/benchmark whether a columnar data-oriented layout like this is better but for now best to keep things consistent.

Reply:
And actually we should rename `OpId` to `SubgraphId` to use consistent language but thats unrelated to this pr

Reply:
Done, will leave the rename change for another PR.